system/hardware/force_cpu_rendering.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/bash
+
+# force CPU rendering on respective platforms
+if [ -f /TICI ]; then
+  export QT_QPA_PLATFORM=eglfs                                    # offscreen doesn't work with EGL/GLES
+  export LD_PRELOAD=/usr/lib/aarch64-linux-gnu/libEGL_mesa.so.0   # force loading mesa driver over adreno
```

Comment:
can we just link against this at build time instead of the normal adreno EGL? I'd like to avoid this wrapper script if possible

Reply:
works if we add a symlink to third_party

Reply:
setting the env variables is still necessary, but can be moved to C++ since we don't need to intercept the LD loading anymore (made Hardware::force_cpu_rendering)